modules/exploits/linux/local/diamorphine_rootkit_signal_priv_esc.rb

```diff
@@ -88,12 +88,6 @@ def check
   end
 
   def exploit
-    if is_root?
-      unless datastore['ForceExploit']
-        fail_with Failure::BadConfig, 'Session already has root privileges. Set ForceExploit to override.'
-      end
-    end
```

Comment:
The `is_root?` check should remain here.

Reply:
Ok, I will be sure to get that added back in.